### PR TITLE
chore: update meta data for thomas and his talk

### PR DESCRIPTION
**Pull request by daKmoR**
- add talk description

### CODE REVIEW

File: server/schema/keywords.ts [hunk 1]
```diff
@@ -41,6 +41,7 @@ enum Keyword {
   RESCRIPT= "ReScript",
   RUST = "Rust",
   SSR = "SSR",
+  SSG = "Static Site Generator",
```
Comment (bebraw) [inline server/schema/keywords.ts:44]:
```suggestion
  SSG = "Static Site Generation",
```

File: content/react-finland-2022/schedules.ts [hunk 2]
```diff
@@ -543,8 +543,9 @@ We'll go through what the web can actually accomplish on it's own at the moment,
         {
           type: SessionType.TALK,
           level: Level.INTERMEDIATE,
-          title: "",
-          description: "",
+          title: "Faster content websites via SSR and progressive hydration of Web Components",
+          description: `One of the best performance tips is "do less" - transmit less data and do less processing. There are a million details but we will focus on transmitting less JavaScript.
+            We are going to tackle it by doing more work at build time which means we start with almost 0 JS and only load more as the users interacts with the page.`,
```
Comment (bebraw) [inline content/react-finland-2022/schedules.ts:548]:
```suggestion
            We are going to tackle it by doing more work at build time which means we start with almost zero JavaScript and only load more as the users interacts with the page.`,
```

**Review by bebraw — COMMENTED** [review]
Looks exciting. Hopefully we can merge this soon. :)

File: server/schema/keywords.ts (hunk 1 above)
Comment (daKmoR) [reply server/schema/keywords.ts:44]: it's funny how "bad" this term is defined... I search for it and found
- Static-Site Generating
- Static-Site Generate
- Static Site Generators

I guess it's most prominent in the JAMstack and they call it `Static Site Generators`
https://jamstack.org/generators/

```suggestion
  SSG = "Static Site Generators",
```

Comment (bebraw) [reply server/schema/keywords.ts:44]: Yeah, there's no single answer. We can go with yours.

**Comment by daKmoR** [comment]
@bebraw this can be merge 👍

**Comment by bebraw** [comment]
Cool, thanks a lot. The session is starting to look good.

**Pull request merged**